nikola/filters.py

```diff
@@ -160,36 +160,36 @@ def jpegoptim(infile):
     return runinplace(r"jpegoptim -p --strip-all -q %1", infile)
 
 
-def html_tidy_withconfig(infile):
+def html_tidy_withconfig(infile, executable=r'tidy5'):
```

Comment:
These, as well as the commands, do not need `r'strings`'. Regular `'strings'` are enough.

Also, r'strings' can be concatenated with regular 'strings'. The r'string' is an information for the parser only, and it does not do anything if there are no backslashes in the string literal.